docs/en/concepts-and-designs/oal.md

```diff
@@ -78,10 +78,15 @@ The parameter (2) is the status of this request. The status(success/failure) eff
 **percentile** is the first multiple value metrics, introduced since 7.0.0. As having multiple values, it could be query through `getMultipleLinearIntValues` GraphQL query.
 In this case, `p99`, `p95`, `p90`, `p75`, `p50` of all incoming request. The parameter is the precision of p99 latency calculation, such as in above case, 120ms and 124 are considered same.
 Before 7.0.0, use `p99`, `p95`, `p90`, `p75`, `p50` func(s) to calculate metrics separately. Still supported in 7.x, but don't be recommended, and don't be included in official OAL script. 
-> All_p99 = from(All.latency).p99(10);
+> all_p99 = from(All.latency).p99(10);
 
 In this case, p99 value of all incoming requests. The parameter is the precision of p99 latency calculation, such as in above case, 120ms and 124 are considered same.
 
+- `errorRate`. The error rate of browser traffic (only browser).
```

Comment:
`percent` is for this. I don't think we need this.

Reply:
If the percent is not suitable, then you should consider whether the scope design is correct.

Reply:
And, I can't find this in the real OAL script. please recheck.

Reply:
A new `richPercent` function is provided.